lib/api.js

```diff
@@ -403,6 +389,62 @@ if (process.env.browser) {
 		bundle = bundle || this.bundleName;
 		return this.getDialog(name, bundle).querySelector('iframe').contentWindow.document;
 	};
+
+	if (true || window._ncgHasSounds) {
+		/**
+		 * Plays the sound cue of the provided `cueName` in the current bundle.
+		 * Does nothing if the cue doesn't exist or if the cue has no assigned file to play.
+		 * @param cueName {String}
+		 * @param [opts] {Object}
+		 * @param [opts.updateVolume=true] - Whether or not to let NodeCG automatically update this instance's volume
+		 * when the user changes it on the dashboard.
+		 * @returns {Object} - A SoundJS AbstractAudioInstance.
+		 */
+		NodeCG.prototype.playSound = function (cueName, opts) {
+			opts = opts || {};
+			if (opts.updateVolume === undefined) {
+				opts.updateVolume = true;
+			}
+
+			// Create an instance of the sound, which begins playing immediately.
+			const instance = createjs.Sound.play(cueName);
+			instance.cueName = cueName;
+
+			// Set the volume.
+			this._soundCues.value.some(cue => {
+				if (cue.name === cueName) {
+					_setInstanceVolume.bind(this)(instance, cue);
```

Comment:
Would be better to refactor this to pass in `this` as a parameter rather than create a new function for each cue name in a loop. Negligible performance hit but it's a code smell imo